internal/db/src/schema/rbac.ts

```diff
@@ -54,7 +54,6 @@ export const permissionsRelations = relations(permissions, ({ one, many }) => ({
 export const keysPermissions = mysqlTable(
   "keys_permissions",
   {
-    tempId: bigint("temp_id", { mode: "number" }).autoincrement().notNull(),
```

Comment:
What's the order of merging here?
We need to migrate the db first, right?

Generally when schema changes are involved, please be extra careful about shipping the least amount of changes, cause it's a pain to roll back larger changes if necessary

In this case the perfect PR would be just addressing the column, nothing else